apps/blueprints/src/pages/user/blueprints.tsx

```diff
@@ -27,14 +28,25 @@ export const UserBlueprints: NextPage<UserBlueprintsProps> = ({ blueprints }) =>
           >
             <Link href="/user/blueprint-create">
               <a>
-                <Button colorScheme="green">create blueprint</Button>
+                <Button primary>Create Blueprint</Button>
               </a>
             </Link>
           </Box>
           <Box>
-            {blueprints.map((bp) => (
-              <BlueprintLink key={bp.id} blueprint={bp} editLink type="row" />
-            ))}
+            {blueprints.length === 0 ? (
+              blueprints.map((bp) => (
+                <BlueprintLink key={bp.id} blueprint={bp} editLink type="row" />
+              ))
+            ) : (
+              <p
+                className="empty"
+                css={{
+                  marginTop: 10,
+                }}
+              >
+                You don't have any blueprints yet
```

Comment:
Good idea but I don't like this text. You can also do a search that leads to no results. So a "No results found" would be better.